DataAccess/Public/Include/DataAccess/IServerRepositories.hpp

```diff
@@ -10,26 +11,56 @@
 namespace DataAccess
 {
 /**
- * @brief The IMessagesRepository abstract subclass from IMasterRepository
+ * @brief The IMessagesRepository abstract subclass from IMasterRepository.
  */
 struct IMessagesRepository : IMasterRepository
 {
-    /// Pure Virtual method to get Message History
+    /**
+    * @brief virtual method for getting history. \
+    * @param channelID - channel's id from which we want to get message history.
```

Comment:
I think, in 19th line should be '/'

Reply:
I haven't touched that point, but I can fix it.